pages/sunos/prctl.md

```diff
@@ -1,6 +1,6 @@
 # prctl
 
-> Get or set the resource controls of running processes,.
+> Get or set the resource controls of running processes.
 > Tasks, and projects.
```

Comment:
```suggestion
> Get or set the resource controls of running processes, tasks, and projects.
```
I think that is how it's supposed to be.

Reply:
Yeah, I too thought the same.